File: exercises/csharp/TriviaGame/TriviaGame/GameRunner.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaGame
{
   public class GameRunner
    {

        private static bool notAWinner;

        public static void Main(String[] args)
        {
            Game round = new Game();

            round.Add("Chet");
            round.Add("Pat");
            round.Add("Sue");

            Random rand = new Random(Int32.Parse(args[0]));

            do
            {

                round.Roll(rand.Next(5) + 1);

                if (rand.Next(9) == 7)
                {
                    notAWinner = round.WrongAnswer();
                }
                else
                {
                    notAWinner = round.WasCorrectlyAnswered();
                }



            } while (notAWinner);

        }
    }
}

File: exercises/nodejs/src/game-runner.js
import generator from "random-seed";
import Game from "./game";

const gameRunner = (i) => {
  const game = new Game();

  let hasWonGame = false;

  game.addPlayer("Chet");
  game.addPlayer("Pat");
  game.addPlayer("Sue");

  const random = generator.create(i);

  do {
    game.roll(random.range(5) + 1);

    if (random.range(9) === 7) {
      hasWonGame = game.wasIncorrectlyAnswered();
    } else {
      hasWonGame = game.wasCorrectlyAnswered();
    }
  } while (hasWonGame);
};

export default gameRunner;
